x-pack/plugins/canvas/canvas_plugin_src/functions/additional/index.tsx

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { shapeFunction } from '../../../../../../src/plugins/expression_shape/common';
+
+export const functions = [shapeFunction];
```

Comment:
What is the reason for this additional functions folder? Why not add the shapeFunction from the expression_shape plugin?

Reply:
As you can see, at [this line of code](https://github.com/elastic/kibana/blob/7590fc971b386a5453c34cb10fe6997ba6b24a00/x-pack/plugins/canvas/__fixtures__/function_specs.ts#L14), this `function` is used only for the test purpose [here](https://github.com/elastic/kibana/blob/7590fc971b386a5453c34cb10fe6997ba6b24a00/x-pack/plugins/canvas/common/lib/autocomplete.test.ts#L8). I didn't want to remove tests for `shape` at autocomplete.

Reply:
Let's move this registration into the shape plugin, and update that test to just expect a different function.  

Maybe just make a few demo functions in that test file and use those maybe?

Reply:
I've added new `eslint` rule [here](https://github.com/elastic/kibana/blob/d701f0c42996ff1289dec6cfe3f4c2045e5d0072/.eslintrc.js#L449).

And exported function from [here](https://github.com/elastic/kibana/blob/d701f0c42996ff1289dec6cfe3f4c2045e5d0072/src/plugins/expression_shape/__fixtures__/function_specs.ts#L12).